### PR TITLE
Fixes [8825] but this time for realsies and I tested it

### PR DESCRIPTION
**Pull request by Probe1**
adds 

```

    if(stat & (NOPOWER | BROKEN))
        to_chat(user, "<span class='warning'> The powers off, it's no good. </span>")
        return

```

Fixes https://github.com/d3athrow/vgstation13/issues/8825

### CODE REVIEW

**Comment by ghost** [comment]
Make sure the spit still works, it's a child of grill.

**Comment by Probe1** [comment]
I will test it but it has  `use_power = 0` so it shouldn't care.

**Comment by ghost** [comment]
Aight

**Comment by ghost** [comment]
Thanks probe

**Comment by Probe1** [comment]
Tested, PR works, spit is unaffected.

**Comment by ghost** [comment]
👌

**Comment by 9600bauds** [comment]
> leading space

reeeeeee

File: code/modules/food/cooking_machines.dm [hunk 1]
```diff
@@ -136,6 +136,9 @@ var/global/ingredientLimit = 10
 	else . = ..()
 
 /obj/machinery/cooking/attackby(obj/item/I,mob/user)
+	if(stat & (NOPOWER | BROKEN))
+		to_chat(user, "<span class='warning'> The powers off, it's no good. </span>")
```
Comment (Intigracy) [inline code/modules/food/cooking_machines.dm]: remove space between > and T pls thanks, that'll show up ingame and look bad

powers = power's OR power is

Comment (PJB3005) [inline code/modules/food/cooking_machines.dm]: Nah I don't think it would.

What IS unacceptable is the lack of an apostrophe.

**Pull request merged**